src/main/resources/data/hexcasting/recipe/patchi_book.json

```diff
@@ -8,5 +8,11 @@
       "item": "hexcasting:amethyst_dust"
     }
   ],
-  "book": "hexcasting:thehexbook"
+  "result": {
+    "components": {
+      "patchouli:book": "hexcasting:thehexbook"
+    },
+    "count": 1,
```

Comment:
(Nitpick)
`count: 1` is redundant, you can remove it or simply mark it as resolved if you don't want to